api/app/src/external/fhir/shared/extensions/extension.ts

```diff
@@ -1,17 +1,18 @@
 import { Coding, DocumentReferenceContent, Extension } from "@medplum/fhirtypes";
+import { METRIPORT } from "../../../../shared/constants";
+import { isCommonwellContent } from "../../../commonwell/extension";
+import { BASE_EXTENSION_URL } from "./base-extension";
 import { DeepRequired } from "ts-essentials";
-import { METRIPORT } from "../../../shared/constants";
-import { isCommonwellContent } from "../../commonwell/extension";
-
-const DATA_SOURCE_EXTENSION_URL =
-  "https://public.metriport.com/fhir/StructureDefinition/data-source.json";
 
 // URL is required: https://www.hl7.org/fhir/R4/extensibility.html#Extension.url
 export type MetriportExtension = Omit<Extension, "url" | "valueCoding"> &
   Required<Pick<Extension, "url">> & {
     valueCoding: Omit<Coding, "system" | "code"> & DeepRequired<Pick<Coding, "system" | "code">>;
   };
 
+// TODO #712: create this extension
+const DATA_SOURCE_EXTENSION_URL = `${BASE_EXTENSION_URL}/data-source.json`;
```

Comment:
NABD, but had this at the top as I find a good practice to concentrate those there, easier to make adjustments and tweaks on constants/literals.

Reply:
moved to the top